airbyte-integrations/connectors/source-amazon-seller-partner/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "4.2.4"
+version = "4.2.5"
```

Comment:
Can you run `poetry lock` and update the changelog in `doc/integration/source-amazon.md`?

Reply:
Also bump the version in `metadata.yaml` file.